examples/compressible/skamarock_klemp_nonhydrostatic.py

```diff
@@ -44,22 +44,24 @@
 # Equation
 Tsurf = 300.
 parameters = CompressibleParameters()
-eqns = CompressibleEulerEquations(domain, parameters)
+eqns = CompressibleEulerEquations(domain, parameters, u_transport_option='vector_advection_form')
+print(f'Number of DOFs = {eqns.X.function_space().dim()}')
 
 # I/O
 points_x = np.linspace(0., L, 100)
 points_z = [H/2.]
 points = np.array([p for p in itertools.product(points_x, points_z)])
-dirname = 'skamarock_klemp_nonlinear'
+dirname = 'testing'
 
 # Dumping point data using legacy PointDataOutput is not supported in parallel
 if COMM_WORLD.size == 1:
     output = OutputParameters(
         dirname=dirname,
         dumpfreq=dumpfreq,
-        pddumpfreq=dumpfreq,
+        dump_nc=True,
+        dump_vtus=True,
```

Comment:
I think I'd prefer for these changes not to be made here to avoid confusing things, so it'd be good to revert back to the previous outputting settings